OracleSiebel/OEL/build/multi-stage/readme.md

```diff
@@ -0,0 +1,53 @@
+# Multi-Stage
+
+Here you'll find content to allow you to build Siebel as a set of three containers, each encapsulating a specific aspect of Siebel;
+
+* Siebel Application Interface (SAI)
+* Cloud Gateway (CGW)
+* Siebel Enterprise Server (SES).
+
+Once built, you'll want to add [persistence](../persistence) to the containers to make them fit for final use
+
+At present we're providing content only for Oracle Linux under the OEL folder. We expect to add a Windows folder at a later date.
+
+## OEL
```

Comment:
The correct product name is Oracle Linux. It hasn't been "OEL" or had the word Enterprise in its name for over a decade now.

Reply:
Apologies. Fixed.